pkg/catalog/catalog.go

```diff
@@ -1923,6 +1930,28 @@ func (c *Catalog) PrepareGCUncommitted(ctx context.Context, repositoryID string,
 	}, nil
 }
 
+// TrackPhysicalAddress store physical address under repository partition, key is unique identifier that we keep in
+//
+//	time based order to later delete old ones
+func (c *Catalog) TrackPhysicalAddress(ctx context.Context, repository, physicalAddress string) (string, error) {
+	repo, err := c.getRepository(ctx, repository)
+	if err != nil {
+		return "", err
+	}
+	repoPartition := graveler.RepoPartition(repo)
+	id := xid.New().String()
+	key := kvTrackPrefix + id
```

Comment:
Why can't the physical address (prefixed by the repo) be the key?

Reply:
wanted to sort order to be ascending order - our physical addresses ordered by desc.